Lib/test/test_platform.py

```diff
@@ -176,6 +177,23 @@ def test_uname_cast_to_tuple(self):
         )
         self.assertEqual(tuple(res), expected)
 
+    def test_uname_replace(self):
+        res = platform.uname()
+        new = res._replace(
+            system='system', node='node', release='release',
+            version='version', machine='machine')
+        self.assertEqual(new.system, 'system')
+        self.assertEqual(new.node, 'node')
+        self.assertEqual(new.release, 'release')
+        self.assertEqual(new.version, 'version')
+        self.assertEqual(new.machine, 'machine')
+        # processor cannot be replaced
+        self.assertEqual(new.processor, res.processor)
+
+    def test_uname_deepcopy(self):
+        res = platform.uname()
+        self.assertEqual(copy.deepcopy(res), res)
```

Comment:
```suggestion
    def test_uname_copy(self):
        uname = platform.uname()
        self.assertEqual(copy.copy(uname), uname)
        self.assertEqual(copy.deepcopy(uname), uname)

    def test_uname_pickle(self):
        uname = platform.uname()
        for proto in range(pickle.HIGHEST_PROTOCOL + 1):
            with self.subTest(protocol=proto):
                pickled = pickle.dumps(uname, proto)
                self.assertEqual(pickle.loads(pickled), uname)
```

Reply:
This seems reasonable, but if pickle is meant to be supported, it probably needs to be supported across Python versions (major and minor, ideally). Should it support restoration of a pickle from 3.8 or 3.9.0?